runtime/queries/column_time_range.go

```diff
@@ -19,6 +19,8 @@ type ColumnTimeRange struct {
 	TableName  string
 	ColumnName string
 	Result     *runtimev1.TimeRangeSummary
+	// will be empty for profiling queries
+	PolicyFilter string
```

Comment:
It would be great not to mix up the `ColumnTimeRange` (for modeling) and `MetricsViewTimeRange` (dashboard) APIs. There's not so much code here – can we just duplicate it in `MetricsViewTimeRange` so we keep them separated?